cmd/restic/cmd_key.go

```diff
@@ -110,8 +110,8 @@ func getNewPassword(gopts GlobalOptions) (string, error) {
 	newopts.password = ""
 
 	return ReadPasswordTwice(newopts,
-		"enter password for new key: ",
-		"enter password again: ")
+		"enter password (new): ",
```

Comment:
What about "enter new password"? That would be a bit more consistent to the `ReadPasswordTwice` usage in cmd_init.